ui/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MPL-2.0",
   "engines": {
     "node": "10.16.3",
-    "yarn": ">=1.10.0"
+    "yarn": ">=1.17.3"
```

Comment:
Looks like the packages in package.json are more or less up-to-date. I still see the benefit of adding `"yarn": ">=1.17.3"` though. This means we probably won't have any changes in `yarn.lock` and `ui/yarn.lock`.

Reply:
I will check again